CHANGELOG.md

```diff
@@ -1,79 +1,12 @@
- - Non-script version of DuckDuckGo when blocking scripts is not loaded from context menu ([#3285](https://github.com/brave/browser-laptop/issues/3285))
- - Brave can't handle URLs with whitespace in it ([#3167](https://github.com/brave/browser-laptop/issues/3167))
- - Whitespace in url fixes #3167 ([#3168](https://github.com/brave/browser-laptop/issues/3168))
- - Flash on http://www.y8.com/games/superfighters does not work ([#3028](https://github.com/brave/browser-laptop/issues/3082))
- - Handle mergeable entry after adding([#3373](https://github.com/brave/browser-laptop/issues/3373))
- - Clear Browsing Data button on about:preferences not working ([#3335](https://github.com/brave/browser-laptop/issues/3335))
- - 0.11.6-beta3 always wants to report a crash -- when normally exiting -- on macOS([#3394](https://github.com/brave/browser-laptop/issues/3394))
- - location.reload() does not reload the page title ([#2798](https://github.com/brave/browser-laptop/issues/2798))
- - Add new tab button mouseover effect not working ([#3297](https://github.com/brave/browser-laptop/issues/3297))
- - Filesystem API cookie not blocked ([#3286](https://github.com/brave/browser-laptop/issues/3286))
- - Make bookmark and history management look nicer ([#441](https://github.com/brave/browser-laptop/issues/441))
- - Clear "Recently Closed" when clearing history ([#3287](https://github.com/brave/browser-laptop/issues/3287))
- - Shields down shows shows gray block on shield items. ([#3307](https://github.com/brave/browser-laptop/issues/3307))
- - Cannot quit from hamburger menu. ([#3310](https://github.com/brave/browser-laptop/issues/3310))
- - 3rd party storage is not fully blocked ([#3214](https://github.com/brave/browser-laptop/issues/3214))
- - Fixed : Address bar input ignored if no default search engine set ([#3254](https://github.com/brave/browser-laptop/issues/3254))([#3275](https://github.com/brave/browser-laptop/issues/3275))
- - Reordered Hamburger Menu with layout mentioned in #1893 ([#3003](https://github.com/brave/browser-laptop/issues/3003))
- - New tab button too high ([#3208](https://github.com/brave/browser-laptop/issues/3208))([#3213](https://github.com/brave/browser-laptop/issues/3213))
- - After Back/Forward navigation, tab shows page URL instead of page title ([#3200](https://github.com/brave/browser-laptop/issues/3200))([#3202](https://github.com/brave/browser-laptop/issues/3202))
- - Unit Test : "Pinning with partitions" test is failing but the functionality works ([#2942](https://github.com/brave/browser-laptop/issues/2942))
- - Application icon is with poor resolution on Linux ([#3229](https://github.com/brave/browser-laptop/issues/3229))([#3230](https://github.com/brave/browser-laptop/issues/3230))
- - Adblock files should download once, not repeatedly ([#3222](https://github.com/brave/browser-laptop/issues/3222))
+ - Added Ledger Beta!
```

Comment:
these need links to issues like all past change log release notes.
It might also be too bare now but maybe not, just make sure that all added features and bug fixes that might have nagged people in the past are listed.

I'd list bookmarks now show in bookmarks menu, and history now shows in history menu.

I'd also add info that there is a new about:history page (while linking to issues for each).

Example of a good bug to include:
Bookmark title is not updated unbookmarking/rebookmarking

Watch capitalization on Added ledger beta , and I think it's not specific enough to convey what it is to people.  Maybe something like "Added the ability to load BTC funds and micropay sites with the Brave Ledger"